### DISCUSSION TITLE
Fusionner les messages lorsqu'on obtient un objet

### DISCUSSION THREAD

**Discussion by romain22222** [Crownicles/Crownicles#1646] [Nouvelles idées]
Je m'explique : pour le moment, quand on reçoit un objet c'est ainsi : 
![image](https://user-images.githubusercontent.com/29299679/204460414-80d6c06c-4dd1-4731-9b47-a933b36927aa.png)
Les deux embeds sont en 2 messages distincts

Ma suggestion : faire en sorte que les deux embeds soient dans un seul et unique message.

Avantages : 
- Moins de lags du bot (surtout quand on fait beaucoup d'achats, il n'aura qu'un message à envoyer
- Plus de clareté dans les channels bondés (pas de risque de mélange d'embeds relous)
- L'un suit toujours l'autre
- Si une personne fais un rapport dans un endroit non voulu, ce sera plus simple de supprimer les messages.

Inconvénients : 
-... j'en ai pas la

**Comment by BastLast** [comment]
Inconvénients : cest encore du fignolage a lheure ou on doit ajouter des features et des évents, on se penchera la dessus plus tard +je suis quasi sur que ya deja une issue ou une discussion sur le sujet
